testing/bdd/features/validation/sequence_id_num.feature

```diff
@@ -1,6 +1,11 @@
 @validation @syntax @sequence @xsd
 Feature: Sequence ID and Sequence Number
-  Both are mandatory parameters and the sequence number has to be a number (no letters)
+  Both are mandatory parameters and the sequence number has to be a number (no letters).
+  Moreover two documents with the same sequence identifier shall have different sequence numbers.
```

Comment:
The requirement is that two _distinct_ documents shall have different sequence numbers. Somewhere we should probably check that if a document arrives with the same sequence number and it is _identical_ to the previously received document with the same sequence number then it is discarded.